README.md

```diff
@@ -74,13 +78,15 @@ What will the following return?
 
 ```js
 typeof 4
+// undefined // 4
 
 typeof "hello"
-
+// 'string'
 typeof true
+// 'boolean'
 
 2 === 1 || 3 === 4
-
+// 'false'
```

Comment:
Does this return `false` (boolean type) or `"false"` (in quotes, string type)? There is a difference between the two.